lib/app/list.ex

```diff
@@ -0,0 +1,53 @@
+defmodule App.List do
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query
+  alias App.{Item, ListItem, Person, Repo}
+  alias __MODULE__
+
+  schema "lists" do
+    field :name, :string
+
+    belongs_to :people, Person, references: :person_id, foreign_key: :person_id
+    many_to_many(:items, Item, join_through: ListItem)
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(list, attrs \\ %{}) do
+    list
+    |> cast(attrs, [:person_id, :name])
+    |> validate_required([:person_id, :name])
+    |> unique_constraint([:name, :person_id], name: :lists_name_person_id_index)
```

Comment:
What does `name: :lists_name_person_id_index` _do_? 🤷‍♂️ 
Is this enforcing _uniqueness_ for a list name for each person?
This is an _implicit_ requirement. 
I _agree_ that it makes sense, but if it's the case we need to be UI/UX-centric.
How is this handled in the UI? 💭

Reply:
> Is this enforcing uniqueness for a list name for each person?

Yes that's correct. I've defined the unique index on the migration:
https://github.com/dwyl/mvp/blob/68f556225d568e2e3e90ff8092f8041f8fb0b6c5/priv/repo/migrations/20221005142257_add_list.exs#L14-L18

And the schema is using this index to make sure a duplicated list name can't be created by the same person:
![image](https://user-images.githubusercontent.com/6057298/194838493-049d255b-fbb1-46ed-ae48-d2b3c4ba0dae.png)

Again the UI can change and our requirements so this index can be removed/updated. I just build a first version to test the idea of lists

Reply:
I've added the relation between groups and list on #169
The link to the person will be updated with the `permissions` PR